### ISSUE TITLE
Crash during dump with Structure Gel API

### ISSUE THREAD

**Issue by Sunlis** [Prunoideae/ProbeJS#12]
Minecraft: 1.18.2
Forge: 40.1.85

Minimal mod list:
- rhino-forge-1802.2.1-build.255.jar (https://www.curseforge.com/minecraft/mc-mods/rhino/files/4069880)
- architectury-4.10.86-forge.jar (https://www.curseforge.com/minecraft/mc-mods/architectury-api/files/4040966)
- kubejs-forge-1802.5.5-build.562.jar (https://www.curseforge.com/minecraft/mc-mods/kubejs/files/4118107)
- probejs-4.0.0-fix3-forge.jar (https://www.curseforge.com/minecraft/mc-mods/probejs/files/4135593)
- structure_gel-1.18.2-2.4.7.jar (https://www.curseforge.com/minecraft/mc-mods/structure-gel-api/files/4018606)

---

I suspect this is the fault of the [Structure Gel API](https://www.curseforge.com/minecraft/mc-mods/structure-gel-api) mod, but I'm filing here for now in case it can help ProbeJS to be more resilient.

In my custom modpack, running `/probejs dump` resulted in a crash, with the error:
```
[07Dec2022 11:54:57.256] [Server thread/ERROR] [net.minecraft.server.MinecraftServer/]: Encountered an unexpected exception
java.lang.NoClassDefFoundError: com.legacy.structure_gel.core.mixin.ChunkAccessMixin is invalid
```
[latest.log](https://gist.github.com/Sunlis/d138212b749cb0f44fea7f44df53cf18), [crash.txt](https://gist.github.com/Sunlis/41a3e3f3111c9b6b36d889ef95003e7e)

I tried setting up a minimal repro by installing only ProbeJS and Structure Gel API. I was able to repro with this minimal set (full mod list with auto-installed dependencies above). [crash.txt](https://gist.github.com/Sunlis/12520d3f2aaaf99c1a88c47d5c594b6d)

**Comment by Prunoideae** [comment]
This is a known bug due to structure gel refers to mixin class in one of its methods. Which will cause a crash since mixin class is not able to be initialized.

Will push a fix for that later.

**Comment by Prunoideae** [comment]
fixed and released as `4.0.0-fix3` on CurseForge
